src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -338,7 +338,7 @@ private void BuildFallbackBlock(TagHelperContent builder)
 
                 // Build the <meta /> tag that's used to test for the presence of the stylesheet
                 builder
-                    .AppendHtml("<meta name=\"x-stylesheet-fallback-test\" class=\"")
+                    .AppendHtml("<meta property=\"x-stylesheet-fallback-test\" content class=\"")
```

Comment:
I **think** he meant be well-formed XML, rather than actually adhere to the strict XHTML 1.1 doctype. That is practically impossible in today's web applications and in practice doesn't matter at all. The validator understands the concept of an HTML5 doctype with an XHTML xmlns declaration (like my sample above) that invokes the XML parsing rules, as opposed to SGML. I agree that we should generally emit XHTML rather than SGML.